docker/Dockerfile-netclient-multiarch-userspace

```diff
@@ -0,0 +1,38 @@
+FROM gravitl/go-builder:latest as builder
+# add glib support daemon manager
+WORKDIR /app
+
+
+ENV GO111MODULE=auto
+RUN apk add --update git build-base libmnl-dev iptables libpcap-dev
+COPY . .
+
+RUN GOOS=linux CGO_ENABLED=1 /usr/local/go/bin/go build -tags headless -ldflags="-w -s" -o netclient-app .
+
+WORKDIR /root/
+
+RUN git clone https://git.zx2c4.com/wireguard-go && \
```

Comment:
we should not need wireguard-tools or wg-go in the userspace implementation

Reply:
removed